WORKSPACE

```diff
@@ -297,3 +315,18 @@ http_archive(
     strip_prefix = "clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04",
     url = "http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz",
 )
+
+load("//toolchain:emcc_toolchain_config.bzl", "emsdk_configure")
+
+# Should be configured after loading `clang`.
+emsdk_configure(name = "emsdk")
+
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
+
+closure_repositories()
+
+# Do not use `tf_workspace()` - it interferes with gRPC loaded by Asylo
+# and causes missing `ares.h` error.
```

Comment:
Can you explain this a bit more? Can Asylo fix this?

Reply:
If we use `tf_workspace()`, then TensorFlow will use Cares, and it will interfere with Cares used and patched by Asylo
https://github.com/tensorflow/tensorflow/blob/25a06bc503c7d07ffc5480ac107e3c8681937971/tensorflow/workspace.bzl#L970-L975

Added a comment.